src/deprecates.jl

```diff
@@ -39,3 +39,10 @@ end
 ### Deprecated September 2019
 @deprecate sum(A::AbstractArray, w::AbstractWeights, dims::Int) sum(A, w, dims=dims)
 @deprecate values(wv::AbstractWeights) convert(Vector, wv)
+
+### Deprecated November 2021
+@deprecate stdm(x::RealArray, w::AbstractWeights, m::Real; corrected::DepBool=nothing) std(x, w, mean=m, corrected=corrected) false
```

Comment:
why do we disable export? This is breaking I think

Reply:
No, because it's not exported currently. We still overload the function defined in Statistics though to it can be used after `using Statistics`.

Reply:
Ah - OK. I mixed it up with the fact that Statistics exports it.